scout/models/case/case_loading_models.py

```diff
@@ -49,11 +49,19 @@
     "peddy_ped",
     "peddy_ped_check",
     "peddy_sex_check",
+    "exe_ver",
+    "smn_tsv",
+    "reference_info",
     "RNAfusion_inspector",
+    "RNAfusion_inspector_research",
     "RNAfusion_report",
     "RNAfusion_report_research",
 ]
 
+CHROMOGRAPH_PATH_SUFFIX_CHECKS = ["autozygous", "coverage", "upd_regions", "upd_sites"]
+
+REVIEWER_PATH_CHECKS = ["alignment", "alignment_index", "catalog", "vcf"]
```

Comment:
```suggestion
REVIEWER_PATH_CHECKS = list(REViewer.model_json_schema().get('properties'))
```
Are they are just forgotten or is there a reason why you wouldn't check if the reference and ref index? Otherwise, again, something like

Reply:
Forgotten, thanks! :)

Reply:
I think you can get the keys directly of you `for key in cls.model_fields:`